docs/rules/lines-around-comment.md

```diff
@@ -1,20 +1,17 @@
-# Enforce empty lines around comments (lines-around-comment)
+# require empty lines around comments (lines-around-comment)
 
 Many style guides require empty lines before or after comments. The primary goal
 of these rules is to make the comments easier to read and improve readability of the code.
 
 ## Rule Details
 
-This rule allows you to specify whether an empty line should be found
-before or after a comment. It can be enabled separately for both block (`/*`)
-and line (`//`) comments, and does not apply to comments that appear on the same
-line as code.
-
+This rule requires empty lines before and/or comments. It can be enabled separately for both block (`/*`) and line (`//`) comments and does not apply to comments that appear on the same line as code.
 
 By default an empty line is required above a block comment,
```

Comment:
Yeah, good catch. Can you combine into one sentence if there are 2 similar like “does not apply” but I am having a hard time putting humpty dumpty back together again here.